packages/javascript/plotlywidget/package.json

```diff
@@ -22,10 +22,8 @@
     "style/*.*"
   ],
   "scripts": {
-    "build": "npm run build:src",
-    "build:src": "rimraf dist && tsc",
+    "build": "webpack",
```

Comment:
The build for this package is, in fact webpack, and running `npm run build` does use the installed webpack in node_modules. So now we have to run `npm install && npm run build` and there are no deprecation warnings :)